projects/distributed-tracing/src/shared/dashboard/widgets/table/table-widget-renderer.component.ts

```diff
@@ -330,7 +330,10 @@ export class TableWidgetRendererComponent
 
   public onModeChange(mode: TableMode): void {
     this.activeMode = mode;
-    this.model.setMode(mode);
+  }
```

Comment:
on mode change is only called in on init now right? we can probably just remove this method, that call and change the template to use `this.model.mode` directly.